inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp

```diff
@@ -951,6 +952,22 @@ TEST_P(IEClassExecutableNetworkGetMetricTest_OPTIMAL_NUMBER_OF_INFER_REQUESTS, G
     ASSERT_EXEC_METRIC_SUPPORTED(EXEC_NETWORK_METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS));
 }
 
+TEST_P(IEClassExecutableNetworkGetMetricTest_SUBGRAPH_METRIC, GetMetricNoThrow) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED();
+    Core ie;
+    Parameter p;
+
+    ExecutableNetwork exeNetwork = ie.LoadNetwork(simpleNetwork, deviceName);
+
+    std::string partition_metric = METRIC_SUBGRAPH_TAG + std::to_string(0) + "_OPTIMAL_NUMBER_OF_INFER_REQUESTS";
+
```

Comment:
such access to metrics does not look user friendly.

Reply:
@apankratovantonp what do you think?

Reply:
It should be just part of internal or Hided API.